src/link_rfu_2.c

```diff
@@ -51,28 +52,28 @@ static void sub_80FA9D0(u16 a0);
 static void sub_80FAA58(void * a0);
 static void sub_80FAA94(u8 taskId);
 static void sub_80FACF0(u8 taskId);
-static void sub_80FB0E8(u32 a0);
+static void GetLinkmanErrorParams(u32 msg);
 static void sub_80FB564(s32 a0);
 static void sub_80FBB74(void);
-static u8 sub_80FBC70(const u8 *a0, u16 a1);
-static void sub_80FBCF8(u32 a0);
+static u8 GetPartnerIndexByNameAndTrainerID(const u8 *trainerName, u16 trainerId);
```

Comment:
ID->Id

Reply:
IMO "ID" should be in all caps because it's like that in-game